gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```

Comment:
Looks like you only ran `gradle wrapper ...` once instead of twice. This isn't enough to upgrade Gradle.

Reply:
You are right.
Also I added timeout because I saw that you did as well. Is that necessary?

Reply:
Sorry, what timeout is that?

Reply:
dunno, I found out in one of your commits, and I did the same as:
https://github.com/relaycorp/awala-testing-jvm/pull/112/files#diff-12277790711b9f7dc1eb1b40d025e36afe07269ea7fa5a5745dff1079facbc61

Reply:
I never change those files by hand. Gradle must've made that change on the second `gradle wrapper ...` call.

Reply:
If `gradle wrapper ...` added it, we should keep it. And I think  that's the case here.